<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simon Says</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <!-- Title & Toggle -->
    <item>
     <layout class="QHBoxLayout" name="titleLayout">
      <item>
       <widget class="QLabel" name="titleLabel">
        <property name="text"><string>Simon Says</string></property>
        <property name="alignment"><set>Qt::AlignCenter</set></property>
        <property name="font">
         <font>
          <family>Sans Serif</family>
          <pointsize>32</pointsize>
          <bold>true</bold>
          <weight>75</weight>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation"><enum>Qt::Horizontal</enum></property>
        <property name="sizeType"><enum>QSizePolicy::Expanding</enum></property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="themeToggle">
        <property name="text"><string>ðŸŒ™ Dark Mode</string></property>
        <property name="minimumSize"><size><width>120</width><height>30</height></size></property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch><verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet"><string notr="true">
QPushButton#themeToggle { background: none; border: none; font: bold 14px; }
QPushButton#themeToggle:hover { color: #2980B9; }
        </string></property>
       </widget>
      </item>
     </layout>
    </item>
    <!-- Button Grid -->
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <property name="horizontalSpacing"><number>20</number></property>
      <property name="verticalSpacing"><number>20</number></property>
      <item row="0" column="0">
       <widget class="QPushButton" name="redButton">
        <property name="minimumSize"><size><width>150</width><height>150</height></size></property>
        <property name="maximumSize"><size><width>150</width><height>150</height></size></property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch><verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet"><string notr="true">
border-radius: 75px;
background-color: #E74C3C;
border: none;
font: bold 18px;
        </string></property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="blueButton">
        <property name="minimumSize"><size><width>150</width><height>150</height></size></property>
        <property name="maximumSize"><size><width>150</width><height>150</height></size></property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch><verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet"><string notr="true">
border-radius: 75px;
background-color: #3498DB;
border: none;
font: bold 18px;
        </string></property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="greenButton">
        <property name="minimumSize"><size><width>150</width><height>150</height></size></property>
        <property name="maximumSize"><size><width>150</width><height>150</height></size></property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch><verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet"><string notr="true">
border-radius: 75px;
background-color: #2ECC71;
border: none;
font: bold 18px;
        </string></property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="yellowButton">
        <property name="minimumSize"><size><width>150</width><height>150</height></size></property>
        <property name="maximumSize"><size><width>150</width><height>150</height></size></property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch><verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet"><string notr="true">
border-radius: 75px;
background-color: #F1C40F;
border: none;
font: bold 18px;
        </string></property>
       </widget>
      </item>
     </layout>
    </item>
    <!-- Start, Progress, Level/Score, Status -->
    <item>
     <widget class="QPushButton" name="startButton">
      <property name="text"><string>Start Game</string></property>
      <property name="minimumHeight"><number>50</number></property>
      <property name="styleSheet"><string notr="true">
QPushButton { background-color: #34495E; color: white; border-radius: 10px; font: bold 16px; }
QPushButton:hover { background-color: #5D6D7E; }
      </string></property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="progressBar">
      <property name="minimumHeight"><number>20</number></property>
      <property name="textVisible"><bool>false</bool></property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="levelScoreLayout">
      <item>
       <widget class="QLabel" name="levelLabel">
        <property name="text"><string>Level: 0</string></property>
        <property name="font"><font><pointsize>14</pointsize><bold>true</bold></font></property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="scoreLabel">
        <property name="text"><string>Score: 0</string></property>
        <property name="font"><font><pointsize>14</pointsize><bold>true</bold></font></property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="statusLabel">
      <property name="text"><string/></property>
      <property name="minimumHeight"><number>30</number></property>
      <property name="alignment"><set>Qt::AlignCenter</set></property>
      <property name="font"><font><pointsize>12</pointsize></font></property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar"/>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>